README.md

```diff
@@ -2,10 +2,27 @@
 
 ## Getting started
 
+### Get the code
+
 ```
 git clone https://github.com/fortran-lang/stdlib
 cd stdlib
+```
+
+### Build with CMake
+
+```
+cmake .
```

Comment:
I'd advocate pointing people in the direction of out of source builds. There is less that can go wrong and surprise you, and cleanup is a snap.

Reply:
I agree. I scratch my head every time I want to clean the top-level directory after an in-source build